src/kakarot/instructions.cairo

```diff
@@ -51,7 +51,7 @@ namespace EVMInstructions {
 
         let is_pc_ge_code_len = is_le(ctx.call_context.bytecode_len, pc);
 
-        if (is_pc_ge_code_len == TRUE) {
+        if (is_pc_ge_code_len != FALSE) {
```

Comment:
according to @spapinistarkware you just need to use `TRUE` as much as possible

Reply:
No no, the other way around. Avoid TRUE(1), use only FALSE(1)